### PR TITLE
SeedLang docs and examples

### PR DESCRIPTION
**Pull request by wixette**
* Add SeedCalc to the overview doc and diagram.
* Add simple examples scripts for SeedCalc and SeedPython.
* Add a .Net example app to demonstrate how to embed SeedLang in.
* Re-structure the docs and add a "Getting Started" doc.

### CODE REVIEW

File: docs/building_and_testing.md [hunk 1]
```diff
@@ -16,7 +16,6 @@ dotnet build
 To make a release build:
 
 ```shell
-cd csharp
```
[the remark this answers is not in the document]
Comment (wixette) [reply docs/building_and_testing.md:19]: Done

File: docs/getting_started.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+You can also run a script file directly:
+
+```shell
+dotnet run --project src/SeedLang.Shell -- -l SeedCalc -f ../examples/seedcalc/scripts/arithmetic.calc
+```
+
+Or, run the script file with all the shell-based visualizers on:
+
+```shell
+cd SeedLang/csharp
```
[the remark this answers is not in the document]
Comment (wixette) [reply docs/getting_started.md]: Done

File: docs/getting_started.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+dotnet run --project src/SeedLang.Shell -- -l SeedCalc
+```
+
+You can also run a script file directly:
+
+```shell
+dotnet run --project src/SeedLang.Shell -- -l SeedCalc -f ../examples/seedcalc/scripts/arithmetic.calc
+```
+
+Or, run the script file with all the shell-based visualizers on:
```
[the remark this answers is not in the document]
Comment (wixette) [reply docs/getting_started.md:42]: Done. Added a paragraph after the command.

**Review by codingpotato — APPROVED** [review]
LGTM

**Pull request merged**